### ISSUE TITLE
detach-root

### ISSUE THREAD

**Issue by danjohansson** [bhauman/devcards#140]
When using defcard-om with an atom as state. 
Navigating back and forth between the namespace list and the devcard keeps adding om roots that are not detached when navigating back. 
I tried this as a workaround
```om/IWillUnmount
    (will-unmount [_]
                  (println "unmounting this")
                  (let [pn (.-parentNode (om/get-node owner))]
                    (js/setTimeout (fn [] (om/detach-root pn)) 200)))
and it seems to do the trick.

regards
Dan

**Comment by bhauman** [comment]
Yeah I'm trying to fix this for all cases and this solution doesn't seem to work reliably, this is really hard to fix because we are rending a react tree within a react tree. It's limitation with om's current design.
